src/pages/Login/Login.layout.vue

```diff
@@ -55,21 +55,44 @@
         </div>
       </q-form>
       <div class="login__other_links">
-        <a class="login__forgotten_link" href="mailto:adidperu@gmail.com"
-          >¿Olvidaste tu contraseña?</a
-        >
-        <a class="login__get_account" href="mailto:adidperu@gmail.com">
+        <button @click="openRecoverPasswordModal" class="login__forgotten_link">
+          ¿Olvidaste tu contraseña?
+        </button>
+        <button @click="openRecoverPasswordModal" class="login__get_account">
           Crear cuenta
-        </a>
+        </button>
       </div>
     </main>
   </div>
 </template>
 
 <script setup lang="ts">
 import didLogo from 'src/assets/logos/didperu.svg';
-import { QInputProps } from 'quasar';
+import { QInputProps, copyToClipboard, Dialog, Notify } from 'quasar';
 
+const openRecoverPasswordModal = () => {
+  Dialog.create({
+    title: 'Recuperar contrasenha',
+    message: 'Contenido para mostrar contrasenha',
+    cancel: {
+      textColor: 'primary',
+      color: 'secondary',
+      label: 'Entendido',
+    },
+    ok: {
+      textColor: 'secondary',
+      color: 'primary',
+      label: 'Copiar',
+    },
+  }).onOk(() => {
+    copyToClipboard('adidperu@gmail.com').then(() => {
```

Comment:
I think this can give problems since android / iOs needs special permissions to access to device clipboard, but I'm not sure, anyway in case you have problems after this update, keep in mind that could be a problem of permissions

Reply:
![mstsc_CO70c1N1lc](https://user-images.githubusercontent.com/71513679/204647102-ac14a079-c734-40b3-84c5-5cdac8e3a193.png)